connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectAddArtifactsHandler.scala

```diff
@@ -100,7 +100,15 @@ class SparkConnectAddArtifactsHandler(val responseObserver: StreamObserver[AddAr
       // We do not store artifacts that fail the CRC. The failure is reported in the artifact
       // summary and it is up to the client to decide whether to retry sending the artifact.
       if (artifact.getCrcStatus.contains(true)) {
-        addStagedArtifactToArtifactManager(artifact)
+        if (artifact.path.startsWith(
```

Comment:
Hm, shouldn't we do this at `addStagedArtifactToArtifactManager`?

Reply:
The goal for artifact with `forward_to_fs/{dest_fs_path}` path is just to upload file to cloud storage. Do we still need add the artifact file to artifact manager ? 
If you want , I can add it, it is simple.